.github/workflows/rebase.yaml

```
@@ -7,14 +7,16 @@ jobs:
     name: Rebase
     if: |
```

```suggestion
    if: >-
```
noticed on https://github.com/cirrus-actions/rebase/blob/master/README.md#installation